compiler/packages/babel-plugin-react-compiler/src/Entrypoint/Options.ts

```diff
@@ -98,15 +98,15 @@ export type PluginOptions = {
    * provided rules will skip compilation. To disable this feature (never bailout of compilation
    * even if the default ESLint is suppressed), pass an empty array.
    */
-  eslintSuppressionRules?: Array<string> | null | undefined;
+  eslintSuppressionRules: Array<string> | null | undefined;
```

Comment:
We're already spreading `defaultOptions` when parsing PluginOptions, which would set `eslintSuppressionRules` and `sources` if they were undefined in the `Partial<PluginOptions>` parameter. This type change should be a no-op